sklearn/utils/arrayfuncs.pyx

```diff
@@ -4,37 +4,26 @@ Small collection of auxiliary functions that operate on arrays
 """
 
 cimport numpy as cnp
-import  numpy as np
+import numpy as np
```

Comment:
We can remove the imports for `np`, `cnp` and also `cnp.import_array`, because they are no longer used in this file. This means we can remove the "include_np" from the setup.py. Concretely:

```diff
diff --git a/setup.py b/setup.py
index 4466e0aa1a..5251eec836 100755
--- a/setup.py
+++ b/setup.py
@@ -440,7 +440,7 @@ extension_config = {
     "utils": [
         {"sources": ["sparsefuncs_fast.pyx"], "include_np": True},
         {"sources": ["_cython_blas.pyx"]},
-        {"sources": ["arrayfuncs.pyx"], "include_np": True},
+        {"sources": ["arrayfuncs.pyx"]},
         {
             "sources": ["murmurhash.pyx", join("src", "MurmurHash3.cpp")],
             "include_dirs": ["src"],
diff --git a/sklearn/utils/arrayfuncs.pyx b/sklearn/utils/arrayfuncs.pyx
index bac50ce678..4cdb6d6145 100644
--- a/sklearn/utils/arrayfuncs.pyx
+++ b/sklearn/utils/arrayfuncs.pyx
@@ -3,8 +3,6 @@ Small collection of auxiliary functions that operate on arrays
 
 """
 
-cimport numpy as cnp
-import numpy as np
 from cython cimport floating
 from libc.math cimport fabs
 from libc.float cimport DBL_MAX, FLT_MAX
@@ -12,9 +10,6 @@ from libc.float cimport DBL_MAX, FLT_MAX
 from ._cython_blas cimport _copy, _rotg, _rot
 
 
-cnp.import_array()
-
-
 def min_pos(const floating[:] X):
     """Find the minimum value of an array over positive values
```